src/app/components/flyout/index.html

```diff
@@ -102,7 +102,7 @@
       propertyName="settingsKey"
       isOptional="true"
     >
-      Specifies a unique key for the UI Config Service to retrieve stored settings from a database. The UI Config Service saves configuration settings for users to preserve the width of the flyout. This property accepts <sky-code>string</sky-code> values. This property is for internal Blackbaud use only. For Blackbaud developers, the UI Config Service website describes how to <a href="https://docs.blackbaud.com/ui-config-service-docs/learn/integrations/skyux-components/flyouts">apply sticky settings to flyouts</a>.
+      Specifies a unique key for the UI Config Service to retrieve stored settings from a database. The UI Config Service saves configuration settings for users to preserve the width of the flyout. This property accepts <sky-code>string</sky-code> values. This property is for internal Blackbaud use only. For more information about the UI Config Service, see <a stacheRouterLink="/learn/get-started/advanced/sticky-settings">the sticky settings documentation</a>.
```

Comment:
We should remove the "This property is for internal Blackbaud use only."

Reply:
Good catch. Fixed this locally and will push the change shortly.